ExploreASL_ProcessMaster.m

```diff
@@ -54,10 +54,10 @@
 
     % Optional modules
     % The following are optional extensions of the structural module and not required to run, normally they can be ignored:
-    if isfield(x, 'bRunModule_LongReg') && x.bRunModule_LongReg
+    if isfield(x.modules, 'bRunLongReg') && x.modules.bRunLongReg
```

Comment:
This is ok not to have it inside structural - Henk agreed?

Reply:
Henk actually requested this, I had it the other way before.

Henk:
> `DARTEL` and `LongReg` are separate non-standard modules, so `bRunDARTEL` and `bRunLongReg` should not go under `x.modules.structural` (but perhaps under `x.modules`?)